### DISCUSSION TITLE
Any way to search for a user profile?

### DISCUSSION THREAD

**Discussion by kilgoreandy** [TodoLodo2089/cod-python-api#8] [Q&A] [answered]
Hi, stumbled across this repo late tonight and am so thankful. I am migrating from an older api and was wondering if you had an equal method to Search for a player? 

![image](https://user-images.githubusercontent.com/92450298/197935842-48c222af-917f-4dd0-9ae6-ed9b4a44a1fd.png)

**Comment by aaron-fredrick** [comment]
Could you elaborate your question further as to what type of data you looking to be returned? @kilgoreandy

**Comment by kilgoreandy** [comment, marked as answer]
User can search using 

api.Misc.search() 

It's not in the docs but received the info from the discord server !
